android/sdk/src/main/java/org/jitsi/meet/sdk/audiomode/AudioModeModule.java

```diff
@@ -300,7 +300,6 @@ private boolean updateAudioRoute(int mode) {
             audioManager.setMode(AudioManager.MODE_NORMAL);
             audioManager.abandonAudioFocus(null);
             audioManager.setSpeakerphoneOn(false);
-            audioManager.setMicrophoneMute(true);
```

Comment:
My personal preference is to let version control do that. But I won't fight it to death.